src/cdk/bootstrap.ts

```diff
@@ -366,19 +413,25 @@ export class MiraBootstrap {
             // take a new snapshot because file changed.
             await cd.takeSnapshot(cd.defaultSnapshotFilePath)
           }
-          if ((await this.areStackFilesValid())) {
-            console.info(chalk.cyan('Deploying Stack:'),
-              `(via ${chalk.grey(this.stackFile)})`)
+          if (await this.areStackFilesValid()) {
```

Comment:
Think we do this in app.ts as well, worth maybe noting in an issue ticket to migrate to a separate file.

Reply:
This as well, a linter fix.